content/en/logs/workspaces/sql_reference.md

```diff
@@ -39,12 +39,19 @@ WHERE department = 'Sales' AND name LIKE 'J%' {{< /code-block >}} |
     ELSE 'Standard Order' 
   END AS order_type 
 FROM orders {{< /code-block >}} |
+| `WINDOW` | Performs a calculation across a set of table rows that are somehow related to the current row.                 | {{< code-block lang="sql" >}}SELECT
```

Comment:
```suggestion
| `WINDOW` | Performs a calculation across a set of table rows that are somehow related to the current row.                 | {{< code-block lang="sql" >}}SELECT
```
Just a small comment, the word "somehow" here reads a little vague, could it be removed? Also I don't know the technical context so feel free to leave it too :)